bundle/compliance/cis_azure/rules/cis_5_5/rule.rego

```diff
@@ -0,0 +1,23 @@
+package compliance.cis_azure.rules.cis_5_5
+
+import data.compliance.lib.common
+import data.compliance.policy.azure.data_adapter
+
+finding = result {
+	# No filter, all resources will be checked
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(ensure_sku_valid),
+		{"Resource": data_adapter.resource},
+	)
+}
+
+ensure_sku_tier {
+	data_adapter.resource.sku.tier != "Basic"
+} else = false
+
+ensure_sku_valid = r {
+	data_adapter.resource.sku != null
+	r = ensure_sku_tier
+}
```

Comment:
wouldn't that have the same effect?
```suggestion
ensure_sku_valid {
	data_adapter.resource.sku.tier != "Basic"
} else = false
```

Reply:
If there is no `sku` field then it will fail instead of `no_eval`.
This is a safe guard.
```
Evgbs-MacBook-Pro:csp-security-policies evgb$ ./opa_darwin_arm64 build -b ./bundle -e ./bundle/compliance
Evgbs-MacBook-Pro:csp-security-policies evgb$ ./opa_darwin_arm64 test -b ./bundle
bundle/compliance/cis_azure/rules/cis_5_5/test.rego:
data.compliance.cis_azure.rules.cis_5_5.test_not_evaluated: FAIL (144.917µs)
--------------------------------------------------------------------------------
PASS: 844/845
FAIL: 1/845
```